src/app/assets/languages/fr.json

```
@@ -0,0 +1,1011 @@
+    "skip-to-content": "Passer au contenu",
+    "about-name": "\u00c0 propos",
+    "documentation-name": "Documentation",
+    "close-browser-message": "Fermez le navigateur ou ouvrez une fen\u00eatre incognito pour vous connecter \u00e0 nouveau.",
+    "dark-theme": "Th\u00e8me sombre",
+    "logout": "D\u00e9connecter"
+  },
+  "welcome-page": {
+    "welcome-title": "Bienvenue sur le serveur {{98,114,97,110,100,110,97,109,101}",
+    "current-version": "Version actuelle",
+    "latest-version": "Derni\u00e8re version",
+    "already-latest-version": "D\u00e9j\u00e0 \u00e0 jour",
+    "description1": "{{brandname}} est un magasin de donn\u00e9es en m\u00e9moire qui offre l'\u00e9volutivit\u00e9, l'interop\u00e9rabilit\u00e9 et la tol\u00e9rance aux d\u00e9fauts.",
+    "description2": "Utilisez des capacit\u00e9s de cache haute performance et cr\u00e9ez des grilles de donn\u00e9es distribu\u00e9es con\u00e7ues pour les architectures de microservice.",
+    "license": "Disponible sous",
+    "apache-license": "Licence Apache 2.0",
+    "blog": "Blog",
+    "blog-description": "Les derniers articles et nouvelles d'Infinispan.",
+    "connect": "Connectez-vous aux groupes {{brandname}} avec des clients distants.",
+    "download": "T\u00e9l\u00e9charger les clients",
+    "servers": "Installer et configurer les clusters {{brandname}}.",
+    "learn-more": "Commencer",
+    "develop": "Applications de code pour diff\u00e9rents cas d'utilisation.",
+    "tutorials": "Ex\u00e9cuter des tutoriels rapides",
+    "go-to-console": "Ouvrir la console"
+  },
+  "login-form": {
+    "login-main": "Connectez-vous \u00e0 {{brandname}}.",
+    "login-label": "{{brandname}} connexion.",
+    "login": "Connectez-vous",
+    "cancel": "Annuler",
+    "dialog-label": "{{brandname}} Ouvrir une session dans la bo\u00eete de dialogue.",
+    "form-username": "Nom d'utilisateur",
+    "help-username": "Saisissez votre nom d'utilisateur.",
+    "form-password": "Mot de passe",
+    "help-password": "Saisissez votre mot de passe."
+  },
+  "not-found-page": {
+    "title": "Page non trouv\u00e9e",
+    "description": "Cette page n'existe plus.",
+    "button": "Retour \u00e0 la page d'accueil"
+  },
+  "not-auth-page": {
+    "title": "Acc\u00e8s non autoris\u00e9",
+    "description": "L'utilisateur connect\u00e9 n'a pas la permission ADMIN.",
+    "button": "Retour \u00e0 la page d'accueil"
+  },
+  "support": {
+    "no-user": "Pouvoirs requis.",
+    "no-user-text": "Vous devez saisir des identifiants pour acc\u00e9der \u00e0 la console. Cr\u00e9ez un nom d'utilisateur et un mot de passe pour continuer.",
+    "no-user-label": "Aucune identification utilisateur n'existe.",
+    "reload-button": "Recharger",
+    "text-create-user": "Cr\u00e9er un nom d'utilisateur et un mot de passe, puis recharger.",
+    "text-command": "bin/cli.sh user create myuser -p changeme -g admin"
+  },
+  "cache-managers": {
+    "title": "R\u00e9cipient de donn\u00e9es",
```

"Récipient" is more what you would call a bowl of a vase.

I think people would expect a more straightforward translation here: `container` => `conteneur`.

```suggestion
    "title": "Conteneur de donn\u00e9es",
```